README.md

```diff
@@ -21,9 +24,10 @@ able to automate the release process through a Continuous Integration server lik
 
 # Installing deisrel
 
-You can install `deisrel` by installing the latest release from [bintray](https://bintray.com/):
+You can install `deisrel` by installing the latest release:
```

Comment:
probably change this to "You can install the latest `deisrel`..." so they know what version they're getting by clicking on this link?